doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -194,6 +194,9 @@ nRF Desktop
 -----------
 
 * The UUID16 values of GATT Human Interface Device Service (HIDS) and GATT Battery Service (BAS) are moved from advertising data to scan response data.
+* Introduced :ref:`CONFIG_DESKTOP_USB_REMOTE_WAKEUP <config_desktop_app_options>` Kconfig option for :ref:`nrf_desktop_usb_state`.
+  The option enables the USB wakeup functionality in the application.
+  The option selects :kconfig:option:`CONFIG_USB_DEVICE_REMOTE_WAKEUP`.
```

Comment:
I think you can leave out this sentence. The release notes shouldn't contain too many details about the feature

Reply:
This changes API, so maybe we could leave it here to make sure users would update their configurations ?

Reply:
If users enable the functionality only in the stack, it would silently fail.